src/Components/PetContainer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '../Styles/PetContainer.scss';
+import PetCard from './PetCard';
+
+
+const PetContainer = () => {
```

Comment:
This is the component where we will need to bring in data that we iterate through to populate the pet card.

Reply:
Yes maybe we can create some dummy data to iterate through